doc/onboarding/view/step1.md

```diff
@@ -0,0 +1,59 @@
+
+## Step 1/5 : Welcome!
+
+### URI
+
+`:userID/welcome`
+
+### Models
+
+<pre>
+user = {
+	&lt;string&gt; name,
+	&lt;id&gt; id,
+	&lt;email&gt; email,
+	&lt;boolean&gt; is_registered: false
+}
+</pre>
+<pre>
+step = {
+	&lt;string&gt; slug: 'welcome',
+	&lt;step&gt; value: 1/5
+}
+</pre>
+
+### Getters
+
+<pre>
+    getNextStepURI: (state) ->
```

Comment:
I think here is just about getting the next step URI. @gregorylegarec said that the view shouldn't know the next step, all the routing is handled by the ```onboarding``` component. If the user want to go to the next step, there is just a function ```goNext()``` that use the ```onboarding``` component. This latter will run the validating function like to submit a form and redirect to the next step, from outside the current view.

It's just about details but I disagree about immutability here @misstick. For me, immutability is not about using ```getter````/```setter```, it's about using only unchangeable state object. We could also use ```getter``` with mutable object.